devicon.json

```
@@ -7494,6 +7494,27 @@
         "color": "#f37036",
         "aliases": []
     },
+    {
+        "name": "powerpages",
+        "altnames": [
+            "Microsoft Power Pages",
```

Remove trailing comma (not allowed in json code
```suggestion
            "Microsoft Power Pages"
```